### ISSUE TITLE
[a11y Documentation] Post A11y Documentation Audit Work: Landing Page Launch Prep

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#73393]
## CAIA A11ys Assigned
- Sara
- Jamie

## Description
- This ticket is a follow-up/next steps from the A11y Accessibility Documentation Audit and ticket #56000. 
- **Goal:** **Launch** a landing page for accessibility at VA.gov, linking to the content retained from the audit that is still current and/or linking to new pages created in the approved locations to retain content and avoid broken links/potential deletion. [This is currently a WIP in GitHub.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/a11y-landing-page-draft.md) and is ready to be built in the approved environment, when approved. 
- **Identify ownership for documentation:** Martha (OCTO)

## Definition of Done
- [x] **Martha** to talk to Danielle to see where this can go post meeting (1/10/2024)
    - [ ] **Option: Add to Design Website** (primary navigation) with accessibility considerations, tips and resources?
    - [x] **Option: Add to Platform Website** for content relating to process/playbook?
- [ ] **Sara/Jamie and a11ys** can work on identifying the content we should build elsewhere vs. link out.
    - [ ] Assist in creating content (from linked content sources) to make our own - once decision is made for where it lives (i.e. Platform, Design, Confluence, GH etc.) to avoid accidental deletion and/or broken links to valuable content. 
- [ ] Page is approved and launched.

```[tasklist]
### Related Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/56000
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/73320
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/73585
- [x] Confirm that page is crosslinked in design and research sections
- [x] Convert TBI guide to GH page and request link placement
- [ ] Review [presentations](https://vfs.atlassian.net/wiki/spaces/ACP/pages/2167832577/Educational+Materials#Links) and work on converting to GH format
```

**Comment by sara-amanda** [comment]
@artsymartha68 @coforma-jamie please find the follow-up ticket for our next steps post meeting today.

**Comment by coforma-jamie** [comment]
**Updates:** 
- Decision was made to put the new page on the **Platform** websites. As of 2/8/24, Shira is aware and is discussing next steps with the Platform team.
- At Martha's request, I created a [draft of a new VADS accessibility page](https://docs.google.com/document/d/1LvHTm9KHF04nLwqActQ_cIywDgVwr8nETSVHL2Hsg2s/edit?usp=sharing). The page is meant to cover two audiences: those who want to contribute to the VADS (as the page is structured today), and those who are using the VADS and need accessibility guidance. As of 2/8/24, the page was shared in the private accessibility Slack channel and is awaiting feedback.

**Comment by coforma-jamie** [comment]
The VADS accessibility page draft is complete and has been approved by @artsymartha68. I submitted a [PR](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/pull/2523) this morning.

**Comment by coforma-jamie** [comment]
The new [VADS accessibility page](https://design.va.gov/about/accessibility/) is live! We're still waiting on the Platform Content Council to approve and push the new internal-facing accessibility landing page onto Platform, so moving this to blocked for now.

**Comment by coforma-jamie** [comment]
No word from the council yet.

**Comment by coforma-jamie** [comment]
Spoke with Lauren Conrad from the Platform Content team. She says that the reviewing the page is in their backlog, and they'll review it as soon as they can. (see [Slack thread](https://dsva.slack.com/archives/CL181NRJQ/p1711644506559769?thread_ts=1711638036.266569&cid=CL181NRJQ))

**Comment by coforma-jamie** [comment]
Spoke with Lauren Conrad again. She's asked the team to move this up in their backlog to get it online as quickly as possible 🤞

**Comment by coforma-jamie** [comment]
Looks like this is in motion - I was just added to the [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/79528)

**Comment by coforma-jamie** [comment]
The TBI guide has been moved to GH and linked to on the page.

For Josh and Angela's presentations, I:
- Downloaded them to PPTX
- Remediated them
- Uploaded them to the CAIA presentations folder on GH
- Linked to them in our "cookbook," in appropriate places
- Requested that they be added to the live page #83075

The work we can do here is done. I'm closing this ticket. :)
